packages/types/src/internal-msg/popup.ts

```diff
@@ -1,11 +1,13 @@
-import { TxApproval } from './tx-approval';
-import { InternalRequest, InternalResponse, Ping } from './shared';
+import { TransactionView } from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/core/transaction/v1/transaction_pb';
+import { AuthorizeRequest } from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/custody/v1/custody_pb';
+import { Jsonified } from '../jsonified';
+import { InternalMessage, InternalRequest, InternalResponse } from './shared';
 
-type PopupSupportedMsg = TxApproval | Ping; // Add more as needed
```

Comment:
Why are we removing the `Ping` type? Is that not being used anywhere anymore?

Reply:
it isn't used anywhere. i guess that's an excessive change for this, but in particular this was pulled out of a larger refactor PR